packages/server/src/router.ts

```diff
@@ -394,14 +396,11 @@ export class Router<
       Record<TPath, inferProcedureFromOptions<TInputContext, typeof procedure>>,
     TErrorShape
   >;
-  /**
-   * @beta Might change without a major version bump
-   */
-  public subscription<
-    TPath extends string,
-    TInput,
-    TOutput extends Subscription<unknown>,
-  >(path: TPath, procedure: CreateProcedureOptions<TContext, TInput, TOutput>) {
+
+  public subscription(
+    path: string,
+    procedure: CreateProcedureOptions<TContext, any, any>,
+  ) {
```

Comment:
Note: for the implementation, the jsdoc and types are ignored so using `any` a bit more so using `any` and not duplicating the beta tag etc. here and in the equivalents for `query` and `mutation`.